proxy/reverse.go

```diff
@@ -55,6 +57,19 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 	proxyreq := new(http.Request)
 	*proxyreq = *clientreq
 
+	var proxybody []byte
```

Comment:
what i have done before it to wrap the original body with a reset closer. the closer will simply reset the reader offset to 0.